config/locales/inherited_proofing/en.yml

```diff
@@ -62,7 +63,9 @@ en:
     troubleshooting:
       headings:
         missing_required_items: Don’t have a phone number? Here’s how to get help
+        need_assistance: 'Need immediate assistance? Here’s how to get help:'
         need_help_updating: Need to update your information? Here’s how to get help
       options:
+        get_partner_help: Get help at Partner Agency
```

Comment:
Can you confirm that this is meant to say "Partner Agency" and not some form of "VA" ? Typically I see "Partner Agency" used in designs as a placeholder for the actual partner's friendly name, which is then substituted with the partner's `friendly_name`.

https://github.com/18F/identity-idp/blob/c2a3697fbab4a5ed5c0200406f6c2f47995f8834/config/locales/idv/en.yml#L199

Reply:
@aduth What you mentioned is correct. I was using old figma's and confirmed it should say  'Get help at VA.gov'